cob_bringup/drivers/mimic.launch

```diff
@@ -1,11 +1,5 @@
 <?xml version="1.0"?>
 <launch>
-
-	<arg name="rotation" default="0"/>
-
-	<!-- start mimic -->
-	<node pkg="cob_mimic" type="mimic_node.py" name="mimic" cwd="node" respawn="false">
-		<param name="rotation" value="$(arg rotation)"/>
```

Comment:
there's no need for the rotation any more, we'll rotate the ubuntu system settings